decidim-proposals/app/cells/decidim/proposals/proposal_g_cell.rb

```diff
@@ -41,7 +41,11 @@ def metadata_cell_instance
       end
 
       def resource_image_path
-        model.attachments.first&.url
+        return has_image?
+      end
```

Comment:
This may not work as you put it. Try this. 
```suggestion
        return unless model.photo.present?
        
        model.photo.url
      end
```

Reply:
Hi Alex, using `model.photo` fails on this test: `Decidim::Proposals::ProposalGCell show when the proposal has no image renders a placeholder image` with `undefined method `with_attached_file' for an instance of Array` as `model.attachments` is empty array in this case. I modified the `decidim-core/lib/decidim/has_attachments.rb` to account for those cases. LMK what you think.

Reply:
actually this fails due to a wrong spec, in proposal_g_cell_spec.rb you have on line 39 : `before { allow(proposal).to receive(:attachments).and_return([]) }`

which is returning the array causing the issue. If you just allow the factory to follow the correct flow, you will see there is no error.